data/infra/resources/apt_package.yaml

```diff
@@ -135,3 +145,18 @@ examples: |
     options '--no-install-recommends'
   end
   ```
+
+  **Prevent the apt_package resource from installing packages with pattern matching names**:
+
+  By default, the apt_package resource will install the named package.
+  If it can't find a package with the exact same name, it will treat the package name as regular expression string and match with any package that matches that regular expression.
+  This may lead Chef Infra Client to install one or more packages with names that match that regular expression.
+
+  In this example, `anchor_package_regex true` prevents the apt_package resource from installing matching packages if it can't find the `lua5.3` package.
+
+  ```ruby
+  apt_package 'lua5.3' do
+    version '5.3.3-1.1ubuntu2'
```

Comment:
@IanMadd any idea how to turn this off? This isn't even a _range_

Reply:
In this case we can just skip it.